src/components/Area.js

```diff
@@ -734,4 +738,168 @@ Area.Path.defaultProps = {
+      } else {
+        this.str.appendUnattributed(t);
+      }
+    }
+  }
+
+  render(parent, area, p, props, parentStyle = {}) {
+    this.parent = parent;
+    let style = { ...parentStyle, ...this.props.style };
+
+    this.str.free();
+    this.str = new libui.AttributedString('');
+
+    const attrs = Object.keys(style)
+      .map(k => {
+        const v = style[k];
+        switch (k) {
+          case 'color':
+            const color = Color(v);
+            return libui.FontAttribute.newColor(toLibuiColor(Color(v)));
+          case 'fontSize':
+            return libui.FontAttribute.newSize(Number(v));
+          case 'fontFamily':
+            return libui.FontAttribute.newFamily(v);
+          case 'backgroundColor':
+            return libui.FontAttribute.newBackgroundColor(
+              toLibuiColor(Color(v))
+            );
+          case 'fontStyle':
+            if (v in libui.textItalic) {
+              return libui.FontAttribute.newItalic(libui.textItalic[v]);
+            }
+            break;
+          case 'fontWeight':
+            if (typeof v === 'string' && v in libui.textWeight) {
+              return libui.FontAttribute.newWeight(libui.textWeight[v]);
+            } else if (
+              Number(v) >= libui.textWeight.minimum &&
+              Number(v) <= libui.textWeight.maximum
+            ) {
+              return libui.FontAttribute.newWeight(Number(v));
+            }
+            break;
+          case 'textStretch':
+            if (v in libui.textStretch) {
+              return libui.FontAttribute.newStretch(libui.textStretch[v]);
+            }
+            break;
+          case 'textUnderline':
+            if (v in libui.textUnderline) {
+              return libui.FontAttribute.newUnderline(libui.textUnderline[v]);
+            }
+            break;
+          case 'textUnderlineColor':
+            if (v !== 'custom' && v in libui.textUnderlineColor) {
+              return libui.FontAttribute.newUnderlineColor(
+                libui.textUnderlineColor[v]
+              );
+            } else {
+              return libui.FontAttribute.newUnderlineColor(
+                libui.textUnderlineColor[v],
+                toLibuiColor(Color(v))
+              );
+            }
+        }
+      })
+      .filter(x => x);
+
+    this.children.forEach(v => {
+      if (typeof v === 'string') {
+        this.appendText(v, ...attrs);
+      } else {
+        v.render(this, area, p, props, style);
+      }
+    });
+
+    if (!(this.parent instanceof AreaText)) {
+      let textAlign;
+      switch (style.textAlign || 'left') {
+        case 'left':
+          textAlign = libui.textAlign.left;
+          break;
+        case 'center':
+          textAlign = libui.textAlign.center;
+          break;
+        case 'right':
+          textAlign = libui.textAlign.right;
+          break;
+      }
+
+      const font = new libui.FontDescriptor(
+        'Arial',
+        14,
```

Comment:
> Looks like this is done. I'll merge if that's good with you @mischnic

👍 

You might want to change this to `12` for the default font (that's also the default font size of the font button on macOS).